Update Learn fundamental layout comprehension (#41362)
mdn/content: files/en-us/learn_web_development/core/css_layout/fundamental_layout_comprehension/index.md

sidebar: learnsidebar
---

{{PreviousMenuNext("Learn_web_development/Core/CSS_layout/Test_your_skills/Grid", "Learn_web_development/Core/CSS_layout/Responsive_Design", "Learn_web_development/Core/CSS_layout")}}

> This challenge will test your knowledge of the layout features we've covered so far in the module, namely flexbox, floats, grid, and positioning. By the end you will have developed a webpage layout using a variety of techniques.

Reviewer: Maybe

```suggestion
This challenge will test your knowledge of the layout features we've covered so far in the module, namely flexbox, floats, grid, and positioning. By the end you will have developed a webpage layout using most of these fundamental tools.
```

Author: I updated it to "all", as it uses all four of them.

5. The photographs should display as a two-column grid with equal-sized columns and a 5-pixel gap between the images.

## Hints and tips

- You don't need to edit the HTML to complete this challenge.
- There are a few ways to achieve some of the tasks in the project brief, and there often isn't a single right or wrong way to do things. Try a few different approaches and see what works best. Make notes as you experiment.

## Example

The following screenshot shows an example of what the finished layout for the design should look like:

![Finished layout task website. The elements are laid out neatly. There is a website title, above a black nav bar containing 5 equally spaced links. Below the nav bar, there are two sections. On the left there is a blog post: A blog post title followed by the post content. The blog content wraps around a photo that is flush left. On the right side there is a 'photography' title about a group of images laid out in a two-image wide grid.](layout-task-complete.png)

<details>

> <summary>Click here to show the solution</summary>

Reviewer: YOu just said there is no one correct solution. So maybe "a"

```suggestion
<summary>Click here to show a solution</summary>
```
